appserver/payara-appserver-modules/notification-email-core/src/main/java/fish/payara/notification/email/GetEmailNotifierConfiguration.java

```diff
@@ -77,4 +78,9 @@ protected String listConfiguration(EmailNotifierConfiguration configuration) {
         columnFormatter.addRow(values);
         return columnFormatter.toString();
     }
-}
\ No newline at end of file
+
+    @Override
+    protected Map<String, Object> getNotifierConfiguration(EmailNotifierConfiguration nc) {
```

Comment:
You've got an UnSupportedOperationException have you missed adding this method implementation?